src/logic/ratings.tsx

```diff
@@ -0,0 +1,204 @@
+// @flow
+import { useEffect, useCallback } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { useNavigation } from "@react-navigation/native";
+import { add, isBefore, parseISO } from "date-fns";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import useFeature from "@ledgerhq/live-common/lib/featureFlags/useFeature";
+import { accountsCountSelector } from "../reducers/accounts";
+import {
+  ratingsModalOpenSelector,
+  ratingsCurrentRouteSelector,
+  ratingsHappyMomentSelector,
+  ratingsDataOfUserSelector,
+} from "../reducers/ratings";
+import {
+  setRatingsModalOpen,
+  setRatingsCurrentRoute,
+  setRatingsHappyMoment,
+} from "../actions/ratings";
+import { languageSelector } from "../reducers/settings";
+import { track } from "../analytics";
+
+const ratingsDataOfUserAsyncStorageKey = "ratingsDataOfUser";
```

Comment:
you could use the settings for simple storage things like that instead of reimplementing a storage logic (I could be wrong as i'm still reading the code and maybe I don't have all the context necessary)

Reply:
I made a separate logic as the ratings are not strictly related to the settings and it is a very distinct and independent feature. I'm open to discuss this topic more in depth to make the best decision